packages/turbo-workspaces/src/utils.ts

```diff
@@ -90,12 +108,8 @@ function getPnpmWorkspaces({
     try {
       const workspaceConfig = yaml.load(fs.readFileSync(workspaceFile, "utf8"));
       // validate it's the type we expect
-      if (
-        workspaceConfig instanceof Object &&
-        "packages" in workspaceConfig &&
-        Array.isArray(workspaceConfig.packages)
-      ) {
-        return workspaceConfig.packages as Array<string>;
+      if (workspaceConfig instanceof Object) {
+        return parseWorkspacePackages({ workspaceConfig });
```

Comment:
`pnpm` doesn't need to share this codepath as `pnpm-workspace.yaml` has it's own schema.